docs/D-Modularity/functions.md

```diff
@@ -91,13 +91,13 @@ Consider a module that receives a flag from another module and performs a calcul
 
 The C language is a procedural programming language.  It supports modular design through function syntax.  Functions transfer control between one another.  When a function transfers control to another function, we say that it ***calls*** the other function.  Once the other function completes its task and transfers control to the caller function, we say that that other function ***returns*** control to its ***caller***. 
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/function.png)
+![function](https://ict.senecacollege.ca//~ipc144/pages/images/function.png)
```

Comment:
The text on this image is also hard to see on dark mode because the background is transparent. Maybe something like this could work:

![function](https://user-images.githubusercontent.com/86167097/140444773-4e42e495-f5e3-4102-a24a-94131824b0df.png)

Reply:
Hey there. Yeah I noticed that as well. Do you know of any way to change the image? The issue is only hard to see on dark mode. Is it possible to just add inline styles to the image if it's a markdown file image tag? 

I can guarantee this isn't the only page with the issue in dark mode.

Reply:
This works, looks great.

Reply:
Thanks!!!